engine/src/test/java/io/camunda/zeebe/engine/state/deployment/FormStateTest.java

```diff
@@ -38,7 +38,7 @@ void shouldStoreForm() {
     formState.storeFormRecord(formRecord);
 
     // when
-    final var maybePersistedForm = formState.findFormByKey(formRecord.getFormKey());
+    final var maybePersistedForm = formState.findFormByKey(formRecord.getFormKey(), "tenant-id");
```

Comment:
🤔 Would you mind changing `tenant-id` to `<default>`. That's the value we use for the default tenant. This makes it easier on us when we start making forms tenant aware 😄 

🔧 Please use a constant for this.